### PR TITLE
102 connect graph

### PR DESCRIPTION
**Pull request by Cozloschi**

### CODE REVIEW

**Comment by Cozloschi** [comment]
@vtemian

File: provisioning/deploy.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@
     - vars.yml
   tasks:
    - name : Clone git repo
-     git: repo={{ project_repo }} dest={{ project_home }}/{{ project_dir }} accept_hostkey=true force=yes version=79-create-screenshot
+     git: repo={{ project_repo }} dest={{ project_home }}/{{ project_dir }} accept_hostkey=true force=yes version=102-connect-graph
```
Comment (vtemian) [inline provisioning/deploy.yml]: Change it back to master.

File: src/dashboard/models.py [hunk 2]
```diff
@@ -9,7 +9,7 @@
 from django.conf import settings
 from django.core.urlresolvers import reverse
 
-from src.criteria.models import IpCriteria
+from src.criteria.models import IpCriteria, HeaderCriteria, SourceCriteria
```
Comment (vtemian) [inline src/dashboard/models.py]: Why did you import those?

File: src/dashboard/views.py [hunk 3]
```diff
@@ -32,6 +32,7 @@ def get(self, request):
         top_providers = self.compute.providers_byresponsetime(
             'presswise.*.*.page_load', 'sortByMaxima'
         )
+        print top_providers
```
Comment (vtemian) [inline src/dashboard/views.py]: Remove unused print statement

**Comment by vtemian** [comment]
Also use `django-compressor` to compress js and scss

**Comment by Cozloschi** [comment]
@vtemian

**Comment by vtemian** [comment]
LGTM

**Pull request merged**